dataservice/api/biospecimen/models.py

```diff
@@ -131,56 +110,43 @@ class Biospecimen(db.Model, Base):
         creator=lambda genomic_file:
         BiospecimenGenomicFile(genomic_file=genomic_file))
 
+    diagnoses = association_proxy(
+        'biospecimen_diagnoses', 'diagnosis',
+        creator=lambda dg: BiospecimenDiagnosis(diagnosis=dg))
+
     biospecimen_genomic_files = db.relationship(BiospecimenGenomicFile,
                                                 backref='biospecimen',
                                                 cascade='all, delete-orphan')
-    diagnoses = db.relationship('Diagnosis', secondary='biospecimen_diagnosis',
-                                backref=db.backref(
-                                    'biospecimens'))
 
 
-def validate_biospecimen(target):
+class BiospecimenDiagnosis(db.Model, Base):
```

Comment:
It's often good to avoid moving around blocks like this unless it's really needed so as to make commits a little cleaner.

Reply:
I had to move BiospecimenDiagnosis below Biospecimen, bc my reference to Biospecimen inside BiospecimenDiagnosis wasn't resolving.